frontend/src/app/user/user.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, OnInit } from '@angular/core';
+import {ResourcesUser, ResourceUser, User, UserEntityService} from '../';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {ActivatedRoute} from '@angular/router';
+import {FormSubmissionDialogComponent} from '../dynamic-form/dynamic-form.component';
+import {DIALOG_STYLE} from '../forms-common/dialog';
+import {MatDialog, MatDialogRef} from '@angular/material';
+
+@Component({
+    selector: 'app-user',
+    templateUrl: './user.component.html',
+    styleUrls: ['./user.component.css']
```

Comment:
Should follow similar styling to the other pages. [People](https://i.imgur.com/f71EpZL.png)

Can do this by adding the `card.css` style here.
```    styleUrls: ['./user.component.css', '../card.css']```